### DISCUSSION TITLE
Warning in a custom button

### DISCUSSION THREAD

**Discussion by msakal20** [segmentio/evergreen#1236] [Q&A] [answered]
Hello, I have a problem with custom button, when I create a new appearance, I have this warning, from it, I can see that the component button takes only 3 values ["default", "minimum", "main"], but appearance work.
![image](https://user-images.githubusercontent.com/84870471/122586758-ba409100-d065-11eb-976f-84ac9d77bb00.png)
The same warning I see in documentation.
![image](https://user-images.githubusercontent.com/84870471/122586790-c3316280-d065-11eb-94b2-e634858645b5.png)

Сan I fix this warning ?

**Comment by akleiner2** [comment, marked as answer]
Good spot out! Honestly, we can loosen up the prop types / typescript types to accept `string` that'll help get rid of these errors (while still maintaining typesafety). Happy to accept PRs!
